bindings/pyroot/cppyy/CPyCppyy/src/Dispatcher.cxx

```diff
@@ -269,7 +269,8 @@ bool CPyCppyy::InsertDispatcher(CPPScope* klass, PyObject* bases, PyObject* dct,
     Cppyy::TCppIndex_t nData = Cppyy::GetNumDatamembers(baseType);
     if (nData) code << "public:\n";
     for (Cppyy::TCppIndex_t idata = 0; idata < nData; ++idata) {
-        if (Cppyy::IsProtectedData(baseType, idata)) {
+        if (Cppyy::IsProtectedData(baseType, idata)
+            && !Cppyy::IsEnumData(baseType, idata)) {
```

Comment:
How is this change related to the rest of the PR?

Reply:
It is not.  I will remove it.

Reply:
It is not.